webrecorder/browsertrix
# fix: Redirect `/orgs` to default path

Fixes https://github.com/webrecorder/browsertrix/issues/2005



### Changes

Redirects `/orgs` to user's default home page.

### Manual testing

1. Log in as regular user
2. Go to `/orgs`. Verify you're redirected to your default org dashboard
3. Log out and log in as superuser
4. Go to `/orgs`. Verify you're redirected to admin dashboard
